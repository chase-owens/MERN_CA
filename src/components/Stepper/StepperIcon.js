import React from 'react';
import Typography from '@material-ui/core/Typography';

const StepperIcon = ({ content }) => {
  return (
    <Typography
      style={{
        textAlign: 'center',
        height: '100%',
        lineHeight: '30px',
        color: '#fff'
      }}
    >
      {content}
    </Typography>
  );
};

export default StepperIcon;
