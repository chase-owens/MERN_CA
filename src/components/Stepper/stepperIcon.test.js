import React from 'react';
import StepperIcon from './StepperIcon';
import { shallow } from 'enzyme';

describe('StepperIcon', () => {
  it('renders without crashing', () => {
    const stepperIcon = shallow(<StepperIcon />);
    expect(stepperIcon).toMatchSnapshot();
  });
});
