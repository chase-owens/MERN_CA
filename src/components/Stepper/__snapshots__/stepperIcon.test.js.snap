// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepperIcon renders without crashing 1`] = `
<WithStyles(Typography)
  style={
    Object {
      "color": "#fff",
      "fontSize": null,
      "height": "100%",
      "lineHeight": "30px",
      "textAlign": "center",
    }
  }
/>
`;
